src/js/demo.js

```diff
@@ -2,12 +2,16 @@
 
 module.exports = (function(){
 	window.addEventListener('message', (e) => {
-		const data = JSON.parse(e.data);
-		if (data.type === 'resize' && data.url && data.height) {
-			const iframesToResize = document.querySelectorAll(`iframe[src="${data.url}"]`);
-			iframesToResize.forEach((iframe) => {
-				iframe.height = data.height;
-			});
+		try {
```

Comment:
Ideally we would have the error reported to sentry for the Origami team to investigate. Component demos shouldn't really be causing errors in this situation, especially `o-` components which are stable and used in production

Reply:
@JakeChampion in `o-ads` we receive "message" events from many third-parties and not all of them adhere to keeping `data` as JSON. 

How would you deal with this in order to avoid unnecessary noise on Sentry. 

I would really like to get rid of these errors in the console, if possible.

Reply:
Perhaps we can change the event listener namespace that the registry demos use to avoid having a namespace collision with o-ads